config/initializers/assets.rb

```diff
@@ -19,3 +25,11 @@
 
 # Prevent default behaviour that adds all non-JS/CSS assets
 Rails.application.config.assets.precompile.delete(Sprockets::Railtie::LOOSE_APP_ASSETS)
+
+# RailsAdmin assets
+Rails.application.config.assets.precompile << lambda do |filename, path|
+  path =~ /rails_admin/ && !%w(.js .css).include?(File.extname(filename))
+end
+Rails.application.config.assets.precompile << lambda do |filename, path|
```

Comment:
Unused block argument - `filename`. If it's necessary, use `_` or `_filename` as an argument name to indicate that it won't be used.